### DISCUSSION TITLE
More Funbox: hex and guid

### DISCUSSION THREAD

**Discussion by asasine** [monkeytypegame/monkeytype#5620] [Ideas] [answered]
Hello, cool website!

It would be fun to have more programmer funboxes.
1. Hexadecimal: `0x[a-f0-9]`
3. GUIDs: `[:hex:]{8}(-[:hex:]){3}[:hex:]{12}`

I can go about adding these if they sound reasonable.

**Comment by Miodec** [comment, marked as answer]
Sounds good, go for it.
